experiments/run_recourse_experiment.py

```diff
@@ -433,43 +448,63 @@ def run_face(
         model=model,
     )
 
+    start_time = time.time()
+
     # Generate the paths.
     paths = iterator.iterate_k_recourse_paths(
         poi=poi, max_iterations=max_iterations
     )
 
-    return paths
+    return paths, time.time() - start_time
 
 
 def format_results(
     path_dfs: Sequence[pd.DataFrame],
     run_config: Mapping[str, Any],
+    elapsed_recourse_seconds: float,
     mrmc_clusters: Optional[pd.DataFrame] = None,
-) -> Mapping[str, pd.DataFrame]:
+    mrmc_cluster_seconds: Optional[float] = None,
+) -> Sequence[pd.DataFrame]:
     """Formats the results as DataFrames ready for analysis.
 
     It adds the path_id and step_id keys to the mrmc_paths dataframe. It also
     adds keys from the experiment_utils.format_results() function.
 
     Args:
-        mrmc_paths: The list of path dataframes output by recourse iteration.
-        clusters_df: A dataframe containing cluster information.
+        path_dfs: The list of path dataframes output by recourse iteration.
         run_config: The run config used to generate these results.
+        elapsed_recourse_seconds: The number of seconds used to compute
+            path_dfs.
+        mrmc_clusters: An optional dataframe containing cluster information.
+        mrmc_cluster_seconds: An optional float showing how long cluster
+            generation took.
+
 
     Returns:
-        A mapping from dataframe name to formatted dataframe."""
+        A sequence of formatted results dataframes. The first dataframe is
+        always the newly created experiment_config_df."""
     for i, path_df in enumerate(path_dfs):
         path_df["step_id"] = np.arange(len(path_df))
         path_df["path_id"] = i
     paths_df = pd.concat(path_dfs).reset_index(drop=True)
 
     if mrmc_clusters is not None:
         mrmc_clusters["path_id"] = np.arange(len(mrmc_clusters))
-        return experiment_utils.format_results(
+        (
+            experiment_config_df,
+            *rest_of_results,
+        ) = experiment_utils.format_results(
```

Comment:
This is pretty wonky and hard-to-parse formatting. Can you reformat this to make it more readable?